src/lib/viewers/BaseViewer.js

```diff
@@ -1019,6 +1019,15 @@ class BaseViewer extends EventEmitter {
         return permissions.can_annotate || permissions.can_create_annotations;
     }
 
+    /**
+     * Handles the 'scrolltoannotation' event and calls the annotator scroll method
+     * @param {Object} data
+     * @param {string} data.id Annotation id
+     */
+    handleScrollToAnnotation({ id }) {
+        this.annotator.scrollToAnnotation(id);
```

Comment:
We've been keeping the new annotations loosely coupled via events instead of explicit function calls, should we do the same here?

Reply:
This is a legacy event so we should keep it around.